pyttb/tensor.py

```diff
@@ -220,39 +269,60 @@ def collapse(
         ## Form and return the final result
         return ttb.tensor(B, newshape, copy=False)
 
-    def contract(self, i: int, j: int) -> Union[np.ndarray, tensor]:
+    def contract(self, i1: int, i2: int) -> Union[np.ndarray, tensor]:
```

Comment:
Is there a particular reason to prefer `i1, i2` over `i,j`? I don't have a strong rationale but `i`, `j` looks nicer to me.

Reply:
I was trying to be explicit with the ordering. This follows papers that that move away from i,j,k that is used for 3-way tensors.